### DISCUSSION TITLE
Slow Mac installation

### DISCUSSION THREAD

**Discussion by hmltn-0** [explosion/spaCy#10273] [Help: Installation]
I am trying to install Spacy with pip3 install ‘spacy[apple]’ and it’s very slow. Is this behavior expected or is there something wrong?

**Comment by adrianeboyd** [comment]
It depends on what you mean by "slow". There aren't binary wheels on PyPI for spacy and its dependencies for OSX arm64 (M1) yet, so the first time you install, pip will compile everything from scratch, which takes ~5 minutes. `pip` usually caches the locally compiled wheels so future installs of the exact same versions will be much faster.
